scene/gui/range.cpp

```diff
@@ -120,13 +120,14 @@ void Range::set_min(double p_min) {
 }
 
 void Range::set_max(double p_max) {
```

Comment:
What about `set_min()`? Also, if all three are updated, `_validate_values();` can be removed.

Reply:
Ok, the `_validate_values` method has been removed.